data/logs_model/test/fake_elasticsearch.py

```diff
@@ -75,7 +75,7 @@ def post_doc(url, request):
     def index_delete(url, request):
         index_name_or_pattern = url.path[1:]
         to_delete = []
-        for index_name in docs.keys():
+        for index_name in list(docs.keys()):
```

Comment:
Please file a JIRA for us to revert all these `list(somedict.keys())` changes; they aren't necessary unless we're accessing the list component of `.keys`

Reply:
https://issues.redhat.com/browse/PROJQUAY-771